mongo/integration/index_view_test.go

```diff
@@ -116,7 +116,12 @@ func TestIndexView(t *testing.T) {
 			})
 			mt.Run("wildcard projection", func(mt *mtest.T) {
 				iv := mt.Coll.Indexes()
-				proj := bson.D{{"a", int32(1)}, {"b.c", int32(1)}}
+
+				// Create an index with a wildcard projection document. The "_id: false" isn't needed to create the
```

Comment:
So the `b.c` field wasn't proving anything about the driver? Any idea why we included it in the first place?

Reply:
My understanding is that this isn't showing anything about the driver. It instructs the server to create an index on fields `a` and `b.c`. Having just `a` is enough to test that the driver is correctly forwarding the wildcard projection document.

Not sure why it was included. This test would've started to fail either way because `{a: 1}` would be normalized to `{a: true, _id: false}` on server 4.5.x.